### DISCUSSION TITLE
In version 8.5.0, restful interface endpoints collect exception problems

### DISCUSSION THREAD

**Discussion by WatsonHua** [apache/skywalking#9024] [Q&A] [answered]
In version 8.5.0, the request mode, such as GET/POST, will be included in the collection of restful interface endpoints. However, once an endpoint reports an error, the request mode will be removed in the collection, resulting in inaccurate early warning statistics on the success rate of endpoints. Skywalking believes that there are two different endpoints.Could you please tell me how to solve it? Thanks
![skywalking](https://user-images.githubusercontent.com/37655230/167562515-e2d3f841-cd6a-4309-8aa8-2c586dfefe80.png)

**Comment by wu-sheng** [comment, marked as answer]
So, what is the question?

**Reply by WatsonHua** [reply]
When an endpoint reports an error, want the endpoint to have a request mode ({POST}/{GET}), otherwise the percentage of success of the endpoint is always incorrect

**Reply by wu-sheng** [reply]
This relates to the plugin you are using. In SkyWalking, we don't set strict rules for plugins, because they are contributed from global contributors. If you want to enhance some plugins through a pull request, we could have a deep discussion.
General discussion is not suitable for this case.

**Reply by WatsonHua** [reply]
See, it is the plug-in used that results in enhanced restful endpoints, Thanks
